src/components/clusterWizard/wizardTransition.ts

```diff
@@ -210,7 +213,12 @@ export const getWizardStepClusterStatus = (
 ): Cluster['status'] => {
   const { status } = cluster;
   if (['insufficient', 'pending-for-input'].includes(cluster.status)) {
```

Comment:
To be consistent with the other change:
```suggestion
  if (['insufficient', 'pending-for-input'].includes(status)) {
```